### DISCUSSION TITLE
store vs github what is the difference?

### DISCUSSION THREAD

**Discussion by ghost** [t1m0thyj/WinDynamicDesktop#383] [Q&A]
store vs github what is the difference?
and can you type in readme so people can see this?

**Comment by t1m0thyj** [comment]
See [this issue](https://github.com/t1m0thyj/WinDynamicDesktop/issues/242#issuecomment-701114994) for a detailed answer. The Store and GitHub versions are published from the same codebase and have essentially the same features, so I haven't seen a need to document differences in the readme.

**Reply by ghost** [reply]
you forgot to mention that some scripts does not work on store version for example:
Change Windows App Theme!
